rfcs/text/0000-planning-for-the-dcp.md

```diff
@@ -0,0 +1,211 @@
+
+DCP Product Backlog items  are modeled as:
+- [User Stories](http://agiledictionary.com/277/user-story/) with acceptance criteria
+- [Epics](http://agiledictionary.com/309/epic/) contain a related set of smaller User Stories required to complete an objective.
+- [Spikes](http://agiledictionary.com/209/spike/) adopted from eXtreme Programming to answer design questions.
+  - [Example use in the Scaled Agile Framework](https://www.scaledagileframework.com/spikes/)
+
+#### User Stories
+
+The [*researcher with a keyboard*](https://docs.google.com/drawings/d/1Li9o_uVkEZDZHbkQv86I49FG8I0zGSgHYkKabp7piEs/edit) and [*researcher with a pipette*](https://docs.google.com/drawings/d/1o6I5BLtJuRJ1vUdb7dHCkFbo0XiaxKDBiVAogqnUowU/edit) journeys are the initial source of user stories that  are modeled as epics in the *dcp* repo,  reflecting the narrative flow of the product. Further refinement by the **Product Owners** is required to both develop and  extend the high-level set of user activities. 
+
+Since journeys may vary per persona, personas MUST be identified for an epic by assigning a standard GitHub label such as "keyboard", "pipette", or "road-builder".  An epic MAY support multiple personas.
+
+#### Technical and Operational Requirements
+Product-wide technical or operational requirements are defined by the **Architecture Team** and modeled as epics in the *dcp* repo.
+
+#### Owners
+
+A DCP owner MUST be assigned to each issue or epic in the *Product Backlog* in the *dcp* repo to shepherd and coordinate progress. This owner is responsible for ensuring that related issues are created in component repo(s) and linked to epics in the *dcp* repo.
+
+For example, an implementation task for a specific component resides in its own repo and is linked to the relevant epic in the *dcp* repo.
+
+* No component-specific GitHub labels such as "Data Store" MUST be assigned to an epic in the *dcp* repo. The link documents the relationship.
+
+### Sprints
+
+ **Four week sprints**. There MUST be one DCP Sprint and Sprint Backlog, not a separate Sprint and Sprint Backlog for each component team. Each DCP Sprint starts on Thursday and ends on Wednesday.  
+ 
+Each Sprint MUST be assigned a ZenHub *Milestone* with a due date set by the **Product Owners**.
+
+ All Scrum events are scheduled on the DCP calendar at a regular cadence. For example:
+![](../images/0000-scrum-calendar.png)
+
+### Refinement
+
+**Every two weeks**. There is a one hour meeting for **Product Owners**, **UXR**, and **Technical Leads** to collaborate on adding detail, owners, and order (but not estimates) to the items in the _dcp_ repo *Product Backlog* to reflect DCP priorities and address emerging issues. Dependencies between components are also clearly identified. (Refinement may also occur during the Sprint Review.)
```

Comment:
I can't see that 1 hour every 2 weeks would be anywhere near enough. We can spend more than half an hour just discussing a single issue within techarch.

Reply:
How much time would you propose then? Originally, we recommended 2 hours every 2 weeks.

Reply:
I guess it depends on the level of detail. So I'm happy with suck it and see.

Reply:
In my experience, these meetings go well if there's substantial prep work to make sure that the board is close to what it should be. If we acknowledge this and continuously assign AIs to refine tickets before the meeting, the refinement meeting will have a higher chance of success.